integration-tests/realm-apps/with-custom-function/secrets.json

```diff
@@ -1,3 +1,3 @@
 {
-    "local-mongodb-uri": "mongodb://localhost:26000"
+    "mongodb_uri": "mongodb://localhost:26000"
```

Comment:
Just wondering if the app importer might be relying on `local-mongodb-uri` given these other occurrences:
* [with-jwt/secrets.json](https://github.com/realm/realm-js/blob/9bdd7cc384f3be4c5f4afae40927c692bb7d0a4b/integration-tests/realm-apps/with-jwt/secrets.json#L3)
* [with-db/secrets.json](https://github.com/realm/realm-js/blob/9bdd7cc384f3be4c5f4afae40927c692bb7d0a4b/integration-tests/realm-apps/with-db/secrets.json#L2)
* [with-db-flex/secrets.json](https://github.com/realm/realm-js/blob/9bdd7cc384f3be4c5f4afae40927c692bb7d0a4b/integration-tests/realm-apps/with-db-flx/secrets.json#L2)
* [with-db-flex/services/mongodb/config.json](https://github.com/realm/realm-js/blob/9bdd7cc384f3be4c5f4afae40927c692bb7d0a4b/integration-tests/realm-apps/with-db-flx/services/mongodb/config.json#L11)

Reply:
Actually .. Thinking more about it, it should probably not have been renamed.